rotimi-best/classroomio
# feat: gitpod setup of classroomio

## What does this PR do?



Continues the work on https://github.com/rotimi-best/classroomio/pull/57



## Type of change




## How should this be tested?



- Click on button gitpod setup in README
- It should startup all of classroomio in the browser

## Checklist



### Required


### Appreciated
